### PR TITLE
leksah instructions

### PR DESCRIPTION
**Pull request by JPMoresmau**

### CODE REVIEW

**Comment by JPMoresmau** [comment]
Fixes #142

File: docs/leksah.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Getting started
+
+```bash
+git clone https://github.com/haskell/haskell-ide-engine.git
+cd haskell-ide-engine
+stack install
+```
+
+This will put the `hie` executable into `$HOME/bin/hie`.
```
Comment (rvion) [inline docs/leksah.md]: isn't $HOME/.local/bin the default stack bin path ?

Comment (JPMoresmau) [inline docs/leksah.md]: I've copied that from the emacs instructions...

Comment (cocreature) [inline docs/leksah.md]: Fixed in 988b147b8e8b37981b1f1bf126a8dfdd9ee5d6d4 for emacs & leksah.

**Pull request merged**